crawl-ref/source/dat/des/altar/ecumenical.des

```diff
@@ -419,3 +419,29 @@ x-----x
 x-----x
 xxb+bxx
 ENDMAP
+
+NAME:   dilly_ecumenical_altar_deadguardian
+TAGS:   no_monster_gen transparent
+MONS:   plant
+SUBST: z = t11
+KITEM: E = never_decay human skeleton / gnoll skeleton / elf skeleton / dwarf skeleton
+: dgn.delayed_decay(_G, "e","hoblin skeleton / hobgoblin skeleton / kobold skeleton")
+FTILE: 1et'_  = floor_grass
+: ecumenical_altar_setup(_G)
+MAP
+         t      
+        ttt     
+       tt_t     
+      tt'tt     
+     tt'ztt     
+     ttz'ttt    
+    ttz'zztt     
+   ttzzz'zztt    
+  ttz'''''''tt   
+  tt'xxx+xxx'tt  
+  tt'x.....x'tt  
+  ttex..G..xeet  
+ tteex..E..xett  
+  ttex.x.x.xtt   
+   ttxxx+xxxtt   
```

Comment:
Here the never_decay corpses are entombed, more like a shrine. One way to do it might be this:

```suggestion
   t'xxx+xxx't  
   ttx.....xtt  
   tEm..G..mEt  
   ttx.....xtt  
     xxt+txx   
```